projects/torchaudio/README.md

```diff
@@ -0,0 +1,38 @@
+# Torchaudio
+
+This library is part of the PyTorch project. PyTorch is an open source machine learning framework.
+The aim of torchaudio is to apply PyTorch to the audio domain.
+
+## Build Docker
+
+    $ sudo docker build -t oss-sydr-fuzz-torchaudio.
```

Comment:
Add space between `oss-sydr-fuzz-torchaudio` and `.`

Reply:
fixed